src/sass/_common/common.scss

```diff
@@ -609,6 +609,7 @@ ul.bullet.gr-row > li {
 .foot-note {
     font-size: $FONT_SIZE_XS;
     color: $COLOR_GRAY_SHADE;
+    display: inline-block;
```

Comment:
this class is also used in trading times page, have you checked to make sure adding this doesn't break the styling there?

Reply:
Yeah, I've checked there and it didn't break anything